### PR TITLE
Infra 816 heka throughput monitoring

### PR DESCRIPTION
**Pull request by nathanleiby**
Extracts values from CounterFilter payload string.

### CODE REVIEW

File: lua/filters/counterfilter_to_fields.lua [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    local message = {
+      Timestamp  = read_message("Timestamp"),
+      Type       = msg_type,
+      EnvVersion = read_message("EnvVersion"),
+      Fields     = {},
+      Payload    = payload
+    }
+
+    -- Example payload: "Got 5 messages. 5.3 msg/sec"
+    local total, per_sec = string.match(payload, "Got ([0-9]+) messages. ([.0-9]+) msg/sec")
```
Comment (mohit) [inline lua/filters/counterfilter_to_fields.lua]: Seems like the CounterFilter emits two kinds of messages. Are we just parsing one kind? Which one? Should we document that?

File: lua/filters/counterfilter_to_fields.lua [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    end
+
+    message.Fields['messages_total'] = total
+    message.Fields['messages_per_sec'] = per_sec
+
+    inject_message(message)
+    return 0
+end
+
+function timer_event(ns)
```
Comment (mohit) [inline lua/filters/counterfilter_to_fields.lua]: You don't need this if it's not implemented

Comment (nathanleiby) [inline lua/filters/counterfilter_to_fields.lua]: removed timer_event  5de6ca1

File: lua/filters/counterfilter_to_fields.lua (hunk 1 above)
Comment (nathanleiby) [inline lua/filters/counterfilter_to_fields.lua]: documented that we're only parsing one kind of message 5de6ca1

**Comment by mohit** [comment]
LGTM

**Pull request merged**